docs/user/cli/export.rst

```diff
@@ -98,8 +98,13 @@ The ``--output-format`` argument defines the encoding used for export. Currently
 * ``geojson``, ``json``
 * ``gml`` `Geography Markup Language <http://www.opengeospatial.org/standards/gml>`_
 * ``shp`` ESRI Shapefile
+* ``leaflet`` Export data to a leaflet map and open in the default browser, if possible
 * ``null`` suppress output entirely
 
+.. note::
+
+    The leaflet format is intended for testing and small scale data exploration and visualization only. For production map generation it is highly recommended to use GeoServer. Additionally, this command requires the use of an online browser in order to access online resources.
```

Comment:
minor wording - change to "Additionally, this command requires the use of a browser in order to access online resources."